### ISSUE TITLE
[UI BUG] : Uneven spacing between username and password fields.

### ISSUE THREAD

**Issue by RuthwikReddy09** [WiseVizo/login_system#4]
The spacing between username and password field is more than spacing between password and email field.

Even spacing would improve the UI.

<img width="554" alt="Screenshot 2024-01-16 234804" src="https://github.com/WiseVizo/login_system/assets/126862059/cd6bdd34-8873-47cd-8a95-db88aa35f957">

**Comment by WiseVizo** [comment]
Yeah i just generated frontend pages using chatgpt in html. but i want to switch it to react soon so i don't know if its worth fixing
